tools/Ecoregionalization_workflow/.shed.yml

```diff
@@ -0,0 +1,18 @@
+name: Ecoregionalization Workflow 
```

Comment:
Is that really a workflow?

I guess one word all small letters is allowed

Reply:
done! THANK YOU so much wonder Björn!!!